codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java

```diff
@@ -97,6 +99,14 @@ public JacksonSerializer() {
                 }
             }
         }
+
+    }
+
+    //注册泛化接口对应的实际实现类型
+    public static void registerGenericService(String serviceName, String className) {
```

Comment:
Does it have to be a static method?

Reply:
yes,  or we can't get the reference to `JacksonSerializer`